relay/sources/rudder-server-relay-postinst

```diff
@@ -49,8 +49,10 @@ chgrp rudder-policy-reader /var/rudder/share
 chmod g+s-w /var/rudder/share
 
 # change some directory to rudder owner
+mkdir -p /var/rudder/configuration-repository/shared-files
 chown -R root:rudder /var/rudder/configuration-repository/shared-files
 chmod 2750 /var/rudder/configuration-repository/shared-files
+chmod -R u+rwx,g+rX /var/rudder/configuration-repository/shared-files/*
```

Comment:
should be done on /var/rudder/configuration-repository/shared-files/ since the directoy may be empty, especially on install this fails